src/utl/include/utl/Logger.h

```diff
@@ -289,9 +289,9 @@ class Logger
 
     if (count == max_message_print) {
       logger_->log(level,
-                   "[{} {}-{:04d}] message limit reached, "
-                   "this message will no longer print"
-                       + std::string(spdlog::details::os::default_eol),
+                   FMT_RUNTIME("[{} {}-{:04d}] message limit reached, "
+                               "this message will no longer print"
+                               + std::string(spdlog::details::os::default_eol)),
```

Comment:
@gadfort could we just use \n here?  Are you changing the EOL?

Reply:
This is consistent with all the logging commands have that added except the one that doesn't need it. I think it evaluates to \n, but I don't see any reason to change it since it should then be changed everywhere.